dspace-api/src/main/java/org/dspace/storage/bitstore/DSBitStoreService.java

```diff
@@ -208,7 +208,7 @@ private synchronized static void deleteParents(File file) {
      * @return The corresponding file in the file system, or <code>null</code>
      * @throws IOException If a problem occurs while determining the file
      */
-    protected File getFile(Bitstream bitstream) throws IOException {
```

Comment:
Are you sure? Isn;t there any option how to avoid this?

Reply:
I can wrap it into a class.
I based it on this code, which created an InputStream from a File. https://github.com/dataquest-dev/DSpace/pull/972/files/3c408237bd47c57bfc115f9763afa9ec5247f84f#diff-6cf23bef044a6595d39da0066c064829444069693aa2da828fa725f94c907093R74

Reply:
Ok, I think it is OK.